lib/matplotlib/tests/test_lines.py

```diff
@@ -7,7 +7,7 @@
 import six
 
 import nose
-from nose.tools import assert_true, assert_raises
+from nose.tools import assert_true, assert_raises, assert_raises_regexp
```

Comment:
When are we dropping 2.6? We all know it is a dead release, is there a platform we are holding out for? RHEL5 defaults to python2.4, RHEL6 defaults to python2.6. If somebody wants to install the latest matplotlib, is it not reasonable to expect them to also install a version of python which is <7 years old....?